arangod/Network/NetworkFeature.cpp

```diff
@@ -281,14 +281,16 @@ void NetworkFeature::prepare() {
     ci = &server().getFeature<ClusterFeature>().clusterInfo();
   }
 
-  network::ConnectionPool::Config config(
-      server().getFeature<metrics::MetricsFeature>());
+  auto& metricsFeature = server().getFeature<metrics::MetricsFeature>();
```

Comment:
The NetworkFeature already gets the MetricsFeature as a constructor argument: If you save it, you could avoid the getFeature call here as well.